src/main/configurations/Translate/Configuration_SoapEndpointRouter_BeantwoordVraag_v2.xml

```diff
@@ -0,0 +1,95 @@
+<Module xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../FrankConfig.xsd">
+    <Adapter name="SoapEndpointRouter_BeantwoordVraag_v2"
+        active="${SoapEndpointRouter_BeantwoordVraag_v2.Active}"
+        description="">
+        <Receiver name="SoapEndpointRouter_BeantwoordVraag_v2">
+            <WebServiceListener name="SoapEndpointRouter_BeantwoordVraag_v2" address="${zaakbrug.soap.beantwoord-vraag.v2.endpoint}" soap="false"/>
+            <JdbcErrorStorage
+                name="JdbcErrorStorage"
+                datasourceName="jdbc/${database.instance.name}"
+                slotId="${instance.name}/beantwoordVraag_v2"/>
+        </Receiver>
+        <Pipeline>
+            <Exits>
+                <Exit name="EXIT" state="SUCCESS"/>
+                <Exit name="EXCEPTION" state="ERROR"/>
+            </Exits>
+
+            <WsdlXmlValidatorPipe 
+                name="ValidateInput"
+                wsdl="Common/xsd/Zaak-_Documentservices_1_2/zds0120/berichten/zds0120_beantwoordVraag_zs-dms.wsdl"
+                >
+                <Forward name="success" path="UnwrapSoapMessage"/>
+                <Forward name="failure" path="WsdlValidationSoftFailForwarder"/>
+            </WsdlXmlValidatorPipe>
+
+            <XmlSwitchPipe 
+                name="WsdlValidationSoftFailForwarder"
+                xpathExpression="$SoftFail = true()"
+                >
+                <Param name="SoftFail" value="${zaakbrug.soap.beantwoord-vraag.v2.validation-soft-fail}" type="BOOLEAN"/>
+                <Forward name="true" path="UnwrapSoapMessage" />
+                <Forward name="false" path="InvalidXml" />
+            </XmlSwitchPipe>
+
+            <SoapWrapperPipe 
+                name="UnwrapSoapMessage"
+                storeResultInSessionKey="UnwrapMessageResult"
+                direction="UNWRAP"
+                removeOutputNamespaces="true">
+                <Forward name="success" path="SanitizeSoapAction"/>
+            </SoapWrapperPipe>
+
+            <XsltPipe
+                name="SanitizeSoapAction"
+                xpathExpression="replace($Action, '/', '_')"
+                storeResultInSessionKey="SatitizedSoapAction"
+                >
+                <Param name="Action" sessionKey="SOAPAction"/>
+                <Forward name="success" path="ActionSwitch"/>
+                <Forward name="error" path="EXCEPTION"/>
+            </XsltPipe>
+
+            <XmlSwitchPipe name="ActionSwitch"
+                forwardNameSessionKey="SatitizedSoapAction"
+            />
+            
```

Comment:
Can a pipe for error handling be added for when no handler is found? You can use the "notFoundForwardName" attribute on the XmlSwitchPipe to make that work.